src/components/textfieldSlider/js/TextfieldSlider.js

```diff
@@ -39,6 +38,23 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
                 }
             }
         },
+        contextAwareness: {
+            sliderVariety: {
+                checks: {
+                    nativeHTML: {
+                        contextValue: "{fluid.prefsWidgetType}",
+                        equals: "nativeHTML",
+                        gradeNames: "fluid.textfieldSlider.nativeHTML"
+
+                    },
```

Comment:
We could do something like this (it still requires a separate `fluid.textfieldSlider.nativeHTML` grade but is less verbose):

```
        contextAwareness: {
            sliderVariety: {
                checks: {
                    jQueryUI: {
                        contextValue: "{fluid.prefsWidgetType}",
                        equals: "jQueryUI",
                        gradeNames: "fluid.textfieldSlider.jQueryUI"
                    }
                },
                defaultGradeNames: "fluid.textfieldSlider.nativeHTML"
            }
        },
```

That's maybe a clearer expression of the component intention than registering and using `fluid.prefsWidgetType: "nativeHTML"` - how does that sound?